android/build.gradle

```diff
@@ -75,36 +75,34 @@ dependencies {
   // Google Firebase Services
   if (googleFirebaseServicesEnabled) {
     // Import the BoM for the Firebase platform
-    implementation platform('com.google.firebase:firebase-bom:30.5.0')
+    implementation platform('com.google.firebase:firebase-bom:31.2.1')
     // Add the dependencies for the Crashlytics and Analytics libraries
     // When using the BoM, you don't specify versions in Firebase library dependencies
     implementation 'com.google.firebase:firebase-crashlytics'
     implementation 'com.google.firebase:firebase-crashlytics-ndk'
   }
 
-  implementation 'androidx.annotation:annotation:1.5.0'
-  implementation 'androidx.appcompat:appcompat:1.7.0-alpha01'
-  implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-  implementation 'androidx.fragment:fragment:1.5.4'
-  // Lifecycle is added as a workaround for duplicate classes error caused by some outdated dependency:
+  // This line is added as a workaround for duplicate classes error caused by some outdated dependency:
```

Comment:
What line? You removed lifecycle

Reply:
... and added another line with kotlin below.